CurrencyControl/Model/Currency.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CurrencyControl.Model
+{
+    public class Currency
+    {
+        // public int iD { get; set; }
+        public string currencyCode { get; set; }
```

Comment:
incorrect spelling. And not only here.
Property names should be in camal case and start with a capital letter. 

Please take a look at this page: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members

And fix it everywhere accordingly. Please pay attention to the members and parameter names as well.

Reply:
l have double checked the naming and made the changes. thanks